authserver/api/user.py

```diff
@@ -146,8 +146,8 @@ def delete(self, id: str = None):
                 return self.response_handler.successful_delete_response('User', id, user_obj)
             else:
                 return self.response_handler.not_found_response(id)
-        except Exception:
-            return self.response_handler.not_found_response(id)
+        except Exception as e:
+            return self.response_handler.exception_response(exception_name=type(e).__name__)
```

Comment:
You have a couple options:

1. remove the try-except entirely. The error will bubble up to `handle_errors`.
2. catch the exception, and then raise a custom error (which bubbles up to `handle_errors`).

Personally, I'd go with option 1, for now.